UnityProject/Assets/Scripts/Tool/GeigerCounter.cs

```diff
@@ -57,19 +58,19 @@ public void CycleUpdate()
 		if (node  == null) return;
 		if (node.RadiationNode.RadiationLevel > 1000)
 		{
-			SoundManager.PlayNetworked(Noise[Level.Extreme][RNG.Next(0,3)]);
+			SoundManager.PlayNetworkedAtPos(Noise[Level.Extreme][RNG.Next(0,3)], registerItem.WorldPositionServer, sourceObj: gameObject);
```

Comment:
Needs to be fixed

Reply:
Yeah sorry, was just finishing up the lava land stuff to a reasonable place before coming back to fix this.